README.md

```diff
@@ -1,68 +1,72 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## Git Wired
 
-## Available Scripts
+**Git Wired** is a tool for Developers and Managers to visualize the current state of completeness of a project by individually recognizing the state of each wireframe element. It serves as a simple wireframe visualization of [GitHub Project Cards](https://help.github.com/articles/about-project-boards/) for *features* in relation to one another. It also provides visuals for quickly conveying the completeness of a project, as well as other custom states of features based on the card column. The intention is merely a tool for organization, especially on smaller and simpler projects where communication is less formal and features may be forgotten or overlooked. This app will not allow you to add special and specific characteristics to the elements, such as sizes, fonts, or colors, but expects that this type of information is stored in a project card. This is not intended to be a comprehensive wireframe, but rather a quick visual of the basic structure and the state of completeness.
 
-In the project directory, you can run:
 
-### `npm start`
+#### Check it in action!
+* [Git Wired](https://git-wired.herokuapp.com/)
+* [Git Wired API](https://git-wired-be.herokuapp.com/) (and [Repo](https://github.com/hbellows/git_wired_be))
 
-Runs the app in the development mode.<br>
-Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+## Setup
+1. Fork or clone this repo: `git clone git@github.com:JoelSmith123/git_wired_fe.git <name of your choice>`
+  * Note: This app was created with [Create React App](https://github.com/facebook/create-react-app).
+1. Install Dependencies: `npm install`
+1. Build Application: `npm run build`
+1. To see the app in action locally: `npm start` then go to: `http://localhost:8080/`
+Note this app uses [Fabric.js](https://github.com/fabricjs/fabric.js), which uses a system dependency - [Cairo](https://www.cairographics.org/download/)
```

Comment:
TO DO - Lines 15 & 16 appear on same line. Add punctuation or something to make it look better.